website/data-storage/src/programs/programs.service.ts

```diff
@@ -88,11 +87,11 @@ export class ProgramsService {
     }
   }
 
-  async setStatus(id: string, chain: string, genesis: string, status: InitStatus): Promise<Program> {
+  async setStatus(id: string, genesis: string, status: InitStatus): Promise<Program> {
     return new Promise((resolve) => {
       setTimeout(async () => {
-        console.log(id, chain, genesis, status);
-        let program = await this.findProgram({ id, chain, genesis });
+        console.log(id, genesis, status);
```

Comment:
This string is unnecessary, we can delete it

Reply:
Done.